packages/nimble-components/src/mapping/mention-user/tests/mapping-mention-user.stories.ts

```diff
@@ -0,0 +1,35 @@
+import { html } from '@microsoft/fast-element';
```

Comment:
Make sure to clean-up references to mention in the filename / contents here too

Reply:
Updated